child-logger.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var defaultLevels = require('./default-levels.js');
+var makeLogMethod = require('./log-method');
+var errors = require('./errors');
+
+module.exports = ChildLogger;
+
+function ChildLogger(config) {
+    this.mainLogger = config.mainLogger;
+    this.path = config.path;
```

Comment:
I'd recommend the application developers manages references.

Doing this for them encourages singletons and does not allow us to gaurd against namespace collisions.